src/mesh/wifi/WiFiServerAPI.h

```diff
@@ -44,7 +44,7 @@ class WiFiServerPort : public WiFiServer, private concurrency::OSThread
     WiFiServerAPI *openAPI = NULL;
 
   public:
-    WiFiServerPort();
+    WiFiServerPort(int port);
```

Comment:
src/mesh/wifi/WiFiServerAPI.h:47: [low:style] Class 'WiFiServerPort' has a constructor with 1 argument that is not explicit. [noExplicitConstructor]